src/main/java/frc/robot/commands/outtake/StopOuttakeCommand.java

```diff
@@ -25,6 +25,11 @@ public void execute() {
     outtake.setSpeed(0);
   }
 
+  @Override
+  public void end(boolean interrupted) {
```

Comment:
Stop command does not need to set the motor speed to zero in end() because the speed is already being set to 0 in execute. end() can be removed here